service/rpc/rpc.go

```
@@ -192,9 +196,39 @@ func newClientForIPFS(projectID, projectSecret string, continueOnly bool) *http.
 
 // newHTTPClientForRPC returns an http.Client configured with default settings intended for RPC calls.
 func newHTTPClientForRPC(continueTrace bool, spanOptions ...sentry.SpanOption) *http.Client {
+	// get x509 cert pool
+	pool, err := x509.SystemCertPool()
+	if err != nil {
+		panic(err)
+	}
+
+	// walk every file in the tls directory and add them to the cert pool
+	filepath.WalkDir("_deploy/root-certs", func(path string, d fs.DirEntry, err error) error {
```

will fail silently if this directory does not exist. for production deployments, the roots should have been added to the system cert pool through the dockerfile already so this code will do nothing